pulp_ansible/app/tasks/collections.py

```diff
@@ -431,6 +430,22 @@ async def _add_collection_version(self, api_version, collection_version_url, met
 
         info = metadata["metadata"]
 
+        if self.add_dependents:
+            dependencies = info["dependencies"]
+            tasks = []
+            loop = asyncio.get_event_loop()
+            for name in dependencies.keys():
+                # get everything to avoid adding potential duplicates to pipeline
+                new_req = RequirementsFileEntry(
+                    name=name,
+                    version="*",
+                    source=None,
+                )
+                if not any(map(lambda x: new_req.included(x), self.collection_info)):
+                    self.collection_info.append(new_req)
```

Comment:
Based on our discussion @bmbouter @daviddavis, I decided to get all the versions of a dependency so that the galaxy CLI will have everything it needs to resolve dependencies, a "safe rather than sorry" approach.